deploy/whitelist-crd.yaml

```diff
@@ -1,37 +1,40 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: whitelistitems.security.banzaicloud.com
+    name: whitelistitems.security.banzaicloud.com
 spec:
-  group: security.banzaicloud.com
-  version: v1alpha1
-  names:
-    kind: WhiteListItem
-    plural: whitelistitems
-    singular: whitelistitem
-    shortnames:
-      - wl
```

Comment:
**Question**: why are we dropping `shortnames`?

Reply:
fixed